board/piksiv3/rootfs-overlay/etc/init.d/S84_0_piksi_fpga

```diff
@@ -5,19 +5,8 @@ name="piksi_fpga"
 start() {
   bit="/lib/firmware/piksi_fpga.bit"
   if [ -f "$bit" ]; then
-    tries=1
-    while [ $tries -le 3 ]; do
-      cat "$bit" > /dev/xdevcfg
-      prog_done=`cat /sys/devices/soc0/amba/f8007000.devcfg/prog_done`
-      if [ "$prog_done" -eq "1" ]; then
-        break
-      else
-        echo "ERROR: FPGA configuration failed" | logger
-        echo "ERROR: FPGA configuration failed" | sbp_log --error
-        echo "ERROR: FPGA configuration failed"
-      fi
-      tries=`expr $tries + 1`
-    done
+      modprobe zynq-fpga
+      echo "piksi_fpga.bit" > /sys/class/fpga_manager/fpga0/firmware
```

Comment:
I kept the retry logic as before.
See https://github.com/swift-nav/piksi_buildroot/commit/b5c2bf0e33ea983da9e435906b879730a930d0e8

Reply:
Agree, let's not drop the retry logic yet

Reply:
done